vars/notify.groovy

```diff
@@ -133,6 +133,26 @@ def _sendToAlertlib(subject, severity, body, extraFlags) {
 }
 
 
+// Return the deployer username, ensuring it begins with @ and falling back to
+// the jenkins build user ID if no username is passed.
+String getDeployerUsername(String deployerUsernameParam) {
```

Comment:
Do you want to change build-webapp.groovy to call this?  Seems a shame to have this in a shared location and not use it twice!

I also wonder if deploy-webapp.groovy should use it, but the code there is different and I'm not sure why.

Reply:
Yeah I do want to use this in other locations but I wanted to make sure it was working here first. Can't do a replay test of a job that doesn't exist yet!

Reply:
Sounds good, I'd suggest putting in a TODO in the places you want to use it.